src/pytest/test_replicate.py

```diff
@@ -41,9 +42,9 @@ def testDelReplicate():
   env.skipOnCluster()
 
   master = env.getConnection()
-  slave = env.getSlaveConnection() 
-  env.assertContains("PONG", master.execute_command("ping"))
-  env.assertContains("PONG", slave.execute_command("ping"))  
+  slave = env.getSlaveConnection()
+  env.assertTrue(master.execute_command("ping"))
```

Comment:
@rafie how did it pass without those changes?

Reply:
I think there is some output processing on the part of redis-py in those cases, so it's probably related to the 3.0 <-> 3.4 version differences.

Reply:
Is this PR changes the version we are installing?

Reply:
Yes, we end up with a combination of the latest version of redis-py (3.4.1) with a version of redis-py-cluster (2.1.0) that can handle it (the latter coming from github, not from pypi).
This is currently the state I'm trying to reach in all modules.
The other possibility is redis-py-cluster 2.0 with redis-py 3.0/3.1, which I consider more problematic as it bounds the redis-py version.

Reply:
:+1: